.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,31 @@
-> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.
+# Add here a title for your PR
```

Comment:
why do we nee this? Every PR comes with a title naturally

Reply:
```suggestion
```